src/crl/interactivesessions/InteractiveSessionExecutor.py

```diff
@@ -46,6 +51,8 @@ def __init__(self,
         self.node_ip = node_ip
         self.node_user = node_user
         self.node_password = node_password
+        self.python_command = \
```

Comment:
Please rename to `self.python_alias` or `self.init_env`

Reply:
Renamed to self.init_env